README.md

```diff
@@ -132,13 +132,13 @@ When building a production bundle it is recommended to pass the `--optimize` fla
 
 #### RuntimeOptions (default `undefined`)
 
-This allows you to control aspects of how `elm-make` runs with [GHC Runtime Options](https://downloads.haskell.org/~ghc/7.10.1/docs/html/users_guide/runtime-control.html).
+This allows you to control aspects of how `elm make` runs with [GHC Runtime Options](https://downloads.haskell.org/~ghc/7.10.1/docs/html/users_guide/runtime-control.html).
```

Comment:
Should this link to the latest ghc docs while we're at it? https://downloads.haskell.org/~ghc/latest/docs/html/users_guide/runtime_control.html